### ISSUE TITLE
XP Pen Star 03 used to work out of the box in 18.04

### ISSUE THREAD

**Issue by tinodesigns** [linuxwacom/input-wacom#258]
Hi, 

in Ubuntu 18.04 my XP Pen Star 03 used to work fine with the wacom driver right out of the box. All later versions of Ubuntu (and some other distros I don't have a list of) it is not working anymore. It is recognized in the settings, but says pen not recognized and tablet not recognized. 

It is also not working. 

I was able to get it back to work by adding the following section to my 70-wacom.conf in my new Ubuntu 20.04 install but only the pen is recognized in the settings. Not the tablet.

Section "InputClass"
        Identifier "XP Pen Star 03"
        MatchUSBID "5543:0081"
        MatchProduct "TABLET 1060N"
        MatchDevicePath "/dev/input/event*"
        MatchIsTablet "true"
        Driver "wacom"
EndSection

See screenshots for the missing tablet. It used to recognize the XP Pen Star 03 just fine, right out of the box. 

Is there a fix to this and can you guys make sure newer version of Ubuntu will recognize it right away?

![Bildschirmfoto von 2021-07-06 13-31-52](https://user-images.githubusercontent.com/5517569/124596730-eb410400-de62-11eb-9ce1-fcea162c4441.png)
![Bildschirmfoto von 2021-07-06 13-32-13](https://user-images.githubusercontent.com/5517569/124596735-ec723100-de62-11eb-9753-496c10ac2b7b.png)

I opened a bug at https://bugs.launchpad.net/ubuntu/+source/xf86-input-wacom/+bug/1926860 before. It took me  a while to figure out it was the wacom driver.

Thanks a bunch.

Tino

**Comment by Pinglinux** [comment]
XP pen devices don't run on Wacom kernel driver. input-wacom can not be the root cause of the issue.

**Comment by tinodesigns** [comment]
What do they run on then?
I got it back working when I added "wacom" as driver just the settings for the tablet are missing.

It was confirmed as a xf86-input-wacom package bug on https://bugs.launchpad.net/ubuntu/+source/xf86-input-wacom/+bug/1926860
Is that different from the Wacom kernel driver / input-wacom?

If you close this it would be nice to have at least something that points into the right direction.  

Thanks
